pkgs/by-name/r2/r2modman/package.nix

```
@@ -1,73 +1,65 @@
 {
   lib,
   stdenv,
-  yarn,
+  copyDesktopItems,
   dart-sass,
-  fetchYarnDeps,
-  fixup-yarn-lock,
-  nodejs,
   electron,
   fetchFromGitHub,
-  nix-update-script,
-  makeWrapper,
   makeDesktopItem,
-  copyDesktopItems,
+  makeWrapper,
+  nix-update-script,
+  nodejs,
+  yarn-berry,
 }:
 
 stdenv.mkDerivation (finalAttrs: {
   pname = "r2modman";
-  version = "3.2.10";
+  version = "3.2.11";
 
   src = fetchFromGitHub {
     owner = "ebkr";
     repo = "r2modmanPlus";
-    rev = "v${finalAttrs.version}";
-    hash = "sha256-Z+KCeRTkbHi1vRU/MkkOgaYVQikFkX0OgkncOLsgsQc=";
+    tag = "v${finalAttrs.version}";
+    hash = "sha256-/cnaP5DbEqYEkTPd1QWNArvMMr7UbX/danXa7ISfPpc=";
   };
 
-  offlineCache = fetchYarnDeps {
+  missingHashes = ./missing-hashes.json;
+  offlineCache = yarn-berry.fetchYarnBerryDeps {
+    inherit (finalAttrs) src patches missingHashes;
     yarnLock = "${finalAttrs.src}/yarn.lock";
-    hash = "sha256-V6N0RIjT3etoP6XdZhnQv4XViLRypp/JWxnb0sBc6Oo=";
+    hash = "sha256-XusNho027MjQLaBxaXMOb/2rTiSLmi4mfwY2hjHQZgk=";
   };
 
   patches = [
+    # Temporary fix for MiSide cover image, until https://github.com/ebkr/r2modmanPlus/pull/2024 gets merged
+    ./fix-miside-cover-image.patch
+
     # Make it possible to launch Steam games from r2modman.
     ./steam-launch-fix.patch
   ];
 
   nativeBuildInputs = [
-    yarn
+    copyDesktopItems
     dart-sass
-    fixup-yarn-lock
-    nodejs
     makeWrapper
-    copyDesktopItems
+    nodejs
+    yarn-berry
+    #yarn-berry.yarnBerryConfigHook
```

When it's commented out, the build fails:
```console
       > Running phase: buildPhase
       > patching script interpreter paths in node_modules/
       > find: 'node_modules/': No such file or directory
       > substitute(): ERROR: file 'node_modules/sass-embedded/dist/lib/src/compiler-path.js' does not exist
```

```suggestion
    yarn-berry.yarnBerryConfigHook
```

---

Ah my mistake, going to fix it. I thought it worked without it :)